laravel/framework/2018-03-30-1.yaml

```diff
@@ -0,0 +1,11 @@
+title:     Exploit of encryption failure vulnerability
+link:      https://medium.com/@taylorotwell/laravel-security-release-5-6-15-and-5-5-40-56f1257933a0
+cve:       ~
+branches:
+    5.5.x:
+        time:     2018-03-30 14:15:00
```

Comment:
It seems to me that the commit fixing the issue is laravel/framework@28e53f23a76206fb130e9a54eb95aa3f010e79c9 which was made at 13:26.

Reply:
Yep! You are totally right! I mistakenly reused the "time of announcement" or whatever you want to call it. I just updated this to match the exact commit date. Thanks for the check!